apps/financial-aid/backend/src/app/modules/staff/staff.controller.ts

```diff
@@ -40,10 +41,8 @@ export class StaffController {
     type: StaffModel,
     description: 'Gets staff by nationalId',
   })
-  async getStaffByNationalId(
-    @Param('nationalId') nationalId: string,
-  ): Promise<StaffModel> {
-    const staff = await this.staffService.findByNationalId(nationalId)
+  async getStaffByNationalId(@CurrentUser() user: User): Promise<StaffModel> {
```

Comment:
The endpoint below (need to comment here from Github) is still unguarded (Get(id/:id))

Reply:
That's just the RolesGuard ... which is just checking for a service that's passed by the client.
If I get an authentication token from Veita (which I can do regardless of permission) I'll get the correct service and thus bypass this protection.

Reply:
see it now .. you're right!